src/policy/fees.cpp

```diff
@@ -902,6 +902,14 @@ CFeeRate CBlockPolicyEstimator::estimateSmartFee(int confTarget, FeeCalculation
      * horizons so we already have monotonically increasing estimates and
      * the purpose of conservative estimates is not to let short term
      * fluctuations lower our estimates by too much.
+     *
+     * Note: In scenarios where data is available for higher targets but not
```

Comment:
Nit: The lead is a bit buried here. I would plainly state "Monotonically increasing estimates are not guaranteed in certain data-sparse scenarios. In particular, (fill in the rest of your comment)"

Reply:
Thanks, I'm happy to take this suggestion. What do you think about:

     * Note: Monotonically increasing estimates are not guaranteed in certain
     * data-sparse scenarios. In particular, if estimateCombinedFee returns no
     * valid data (-1) for some estimates for a lower target, but does return
     * valid data for a higher target, the estimate can theoretically return a
     * higher value for higher targets.

Reply:
Great - LGMT

Reply:
OK pushed that in [372ca14](https://github.com/bitcoin/bitcoin/pull/31080/commits/372ca14fbe0e716c789b74b766a8532f0f174268)